catalog-rest-service/src/main/resources/ui/src/constants/services.const.ts

```diff
@@ -34,6 +35,7 @@ export const HIVE = hive;
 export const POSTGRES = postgres;
 export const ORACLE = oracle;
 export const SNOWFLAKE = snowflakes;
+export const GENERIC = generic;
```

Comment:
Use some meaningful name, GENERIC/generic does not clarify the purpose of this icon

Reply:
One suggestion, you can use something like serviceDefault